### PR TITLE
filter campaign by annotators

### PR DESCRIPTION
**Pull request by Qouagga**
#106

### CODE REVIEW

File: backend/tests/views/annotation_campaign.py [hunk 1]
```diff
@@ -85,6 +85,7 @@ def tearDown(self):
     def test_list(self):
         """AnnotationCampaign view 'list' returns list of campaigns"""
         url = reverse("annotation-campaign-list")
+        self.client.login(username="admin", password="osmose29")
```
Comment (ixio) [inline backend/tests/views/annotation_campaign.py:88]: You should also test when the person is logged in and not staff :wink:. (also time to rebase)

Comment (rvovard) [reply backend/tests/views/annotation_campaign.py:88]: New tests are OK for me!

**Review by rvovard — CHANGES_REQUESTED** [review]
Just a really little note.

File: backend/api/migrations/0021_change_zoom_level_format.py [hunk 2]
```diff
@@ -14,6 +14,7 @@ def change_zoom_level(apps, schema_editor):
             spectro.zoom_level = 0
         spectro.save()
 
+
```
[the remark this answers is not in the document]
Comment (Qouagga) [reply backend/api/migrations/0021_change_zoom_level_format.py:17]: In the backend, we use black to harmonize our code. Black add this.

[the remark this answers is not in the document]
Comment (ixio) [reply backend/api/migrations/0021_change_zoom_level_format.py:17]: This has a been fixed some time ago on master :sweat_smile: a simple rebase on master would solve this

**Review by ixio — APPROVED** [review]
I had asked improvements on tests and this looks good to me, gj.

**Pull request merged**